examples/website/plot/plot-layer/axes-layer.ts

```diff
@@ -271,20 +285,20 @@ export default class AxesLayer extends Layer {
       }
     }
 
-    const labels = new Model(gl, {
+    const labels = new Model(gl.device, {
       id: `${this.props.id}-labels`,
       vs: labelVertex,
       fs: labelFragment,
+      bufferLayout: this.getAttributeManager().getBufferLayouts(),
       geometry: new Geometry({
         topology: 'triangle-list',
         attributes: {
           indices: new Uint16Array(labelIndices),
-          positions: new Float32Array(labelPositions),
-          texCoords: {size: 2, value: new Float32Array(labelTexCoords)},
-          normals: new Float32Array(labelNormals)
+          positions: {value: new Float32Array(labelPositions), size: 3},
+          texCoords: {value: new Float32Array(labelTexCoords), size: 2},
+          normals: {value: new Float32Array(labelNormals), size: 3}
```

Comment:
Note: Size now appears to be required – if that isn't an intentional change in v9, I can look into the cause. Requiring it seems fine to me, too.

Reply:
That is intentional, due to WebGPU being more strict about attribute dimensions (e.g. you can't pass 2D array for a `vec3 attribute`)